.travis.yml

```diff
@@ -53,14 +53,15 @@ install:
 script:
   - mkdir -p "$HOME/.ipython/profile_default"
   - "echo 'c.HistoryAccessor.enabled = False\n' > $HOME/.ipython/profile_default/ipython_config.py"
-  - export TEST_EXTRA=""
   - if [[ "$STATIC" == "false" ]]; then
-      if [[ "$COVERAGE" == "true" ]]; then
-        export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-      fi;
       python -c "import numpy; numpy.show_config()";
       python setup.py -q develop;
-      eval py.test nengo -n 2 -v --duration 20 "$TEST_EXTRA";
+      if [[ "$COVERAGE" == "true" ]]; then
```

Comment:
Very minor nitpick, but since I did it to @hunse: could this be the else instead? I.e., Check it coverage is false and do the normal tests, otherwise collect coverage.